hdfs_namenode/tests/fixtures/hdfs_namesystem_info

```diff
@@ -0,0 +1,43 @@
+{
+  "beans" : [ {
+    "name" : "Hadoop:service=NameNode,name=NameNodeInfo",
+    "modelerType" : "org.apache.hadoop.hdfs.server.namenode.FSNamesystem",
+    "Total" : 62725623808,
+    "UpgradeFinalized" : true,
+    "ClusterId" : "CID-44e20313-500c-4587-ad7d-4bf751a4a652",
+    "Version" : "2.7.1, r15ecc87ccf4a0228f35af08fc56de536e6ce657a",
+    "Used" : 24576,
+    "Free" : 55830482944,
+    "Safemode" : "",
+    "NonDfsUsedSpace" : 6895116288,
+    "PercentUsed" : 3.918016E-5,
+    "BlockPoolUsedSpace" : 24576,
+    "PercentBlockPoolUsed" : 3.918016E-5,
+    "PercentRemaining" : 89.00746,
+    "CacheCapacity" : 0,
+    "CacheUsed" : 0,
+    "TotalBlocks" : 0,
+    "TotalFiles" : 1,
+    "NumberOfMissingBlocks" : 0,
+    "NumberOfMissingBlocksWithReplicationFactorOne" : 0,
+    "LiveNodes" : "{\"c68ae3949d7f:50010\":{\"infoAddr\":\"172.24.0.3:50075\",\"infoSecureAddr\":\"172.24.0.3:0\",\"xferaddr\":\"172.24.0.3:50010\",\"lastContact\":2,\"usedSpace\":24576,\"adminState\":\"In Service\",\"nonDfsUsedSpace\":6895116288,\"capacity\":62725623808,\"numBlocks\":0,\"version\":\"2.7.1\",\"used\":24576,\"remaining\":55830482944,\"blockScheduled\":0,\"blockPoolUsed\":24576,\"blockPoolUsedPercent\":3.918016E-5,\"volfails\":0}}",
+    "DeadNodes" : "{}",
+    "DecomNodes" : "{}",
+    "BlockPoolId" : "BP-526874547-172.24.0.2-1574784379919",
+    "NameDirStatuses" : "{\"active\":{\"/hadoop/dfs/name\":\"IMAGE_AND_EDITS\"},\"failed\":{}}",
+    "NodeUsage" : "{\"nodeUsage\":{\"min\":\"0.00%\",\"median\":\"0.00%\",\"max\":\"0.00%\",\"stdDev\":\"0.00%\"}}",
+    "NameJournalStatus" : "[{\"manager\":\"FileJournalManager(root=/hadoop/dfs/name)\",\"stream\":\"EditLogFileOutputStream(/hadoop/dfs/name/current/edits_inprogress_0000000000000000001)\",\"disabled\":\"false\",\"required\":\"false\"}]",
+    "JournalTransactionInfo" : "{\"MostRecentCheckpointTxId\":\"0\",\"LastAppliedOrWrittenTxId\":\"1\"}",
+    "NNStarted" : "Tue Nov 26 16:06:22 UTC 2019",
+    "CompileInfo" : "2015-06-29T06:04Z by jenkins from (detached from 15ecc87)",
+    "CorruptFiles" : "[]",
+    "DistinctVersionCount" : 1,
+    "DistinctVersions" : [ {
+      "key" : "2.7.1",
+      "value" : 1
+    } ],
+    "SoftwareVersion" : "2.7.1",
+    "RollingUpgradeStatus" : null,
+    "Threads" : 33
+  } ]
+}
```

Comment:
This seems to be proper JSON, and I believe `._get_jmx_data()` does expect the response to contain JSON in all cases, so should this be a `.json` file?

Reply:
Yes, that makes text editors happier :)

Reply:
I'd be happy to review that change separately, though, not to clutter this PR.

Reply:
Ok - will post those changes separately

Reply:
@mgarabed Not seeing the change in your latest commit, so to clarify I suppose we can get this one in as JSON? ("That change" in my previous comment was referring to other instances of "plain files that should really be JSON" in test fixtures, which we can indeed address separately.)